src/io_uring/src/lib.rs

```diff
@@ -46,6 +46,12 @@ pub struct IoUring {
     registered_fds_count: u32,
     squeue: SubmissionQueue,
     cqueue: CompletionQueue,
+
+    // Number of ops yet to be submitted.
+    to_submit: u32,
+    // Number of ops yet to be pop-ed from the CQ. These ops either haven't been pop-ed yet,
```

Comment:
This line seems too long.

Reply:
Not sure. I thought comments should be under 100 characters.

Reply:
You are right, this is fine!